diario_bot/twitter.py

```diff
@@ -38,7 +38,7 @@ def split_tweets(found_topics: list, character_limit: int):
     return tweet_list
 
 
-def post_todays_gazette(gazettes: list, dry_run=False):
+def post_gazette(gazettes: list, dry_run=False):
```

Comment:
```suggestion
def post_gazettes(gazettes: list, dry_run=False):
```

Como estamos recebendo uma lista de diários, faz mais sentido deixar o método no plural.